pkg/client/restconfig/restconfig.go

```diff
@@ -77,32 +91,40 @@ func NewRestConfig(timeout time.Duration) (*rest.Config, error) {
 // UpdateQPS modifies a rest.Config to update the client-side throttling QPS and
 // Burst QPS.
 //
-// If Flow Control is enabled on the apiserver, client-side throttling is
-// disabled!
+// If Flow Control is enabled on the apiserver, and client-side throttling is
+// not forced to be enabled, client-side throttling is disabled!
 //
 // If Flow Control is disabled or undetected on the apiserver, client-side
 // throttling QPS will be increased to at least 30 (burst: 60).
 //
+// If client-side throttling is forced to be enabled, no matter whether Flow
+// Control is enabled or not, client-side throttling QPS will be increased to
+// at least 30 (burst: 60).
+//
 // Flow Control is enabled by default on Kubernetes v1.20+.
 // https://kubernetes.io/docs/concepts/cluster-administration/flow-control/
-func UpdateQPS(config *rest.Config) {
+func UpdateQPS(config *rest.Config, forceThrottling bool) {
 	// Timeout if the query takes too long, defaulting to the lower QPS limits.
```

Comment:
Should we rename `forceThrottling` to `forceClientSideThrottling`?

Reply:
The rest.Config should indicate it is a kubernetes client. Plus, I don't think we're able to force throtlling for apiserver, so I pick `forceThrotlling` for short.

Reply:
I see.